tests/multisig/programs/multisig/src/lib.rs

```diff
@@ -17,16 +17,16 @@
 //! the `execute_transaction`, once enough (i.e. `threshold`) of the owners have
 //! signed.
 
-use anchor_lang::accounts::program_account::ProgramAccount;
 use anchor_lang::prelude::*;
 use anchor_lang::solana_program;
 use anchor_lang::solana_program::instruction::Instruction;
 use std::convert::Into;
+use std::ops::Deref;
 
 declare_id!("Fg6PaFpoGXkYsidMpWTK6W2BeZ7FEfcYkg476zPFsLnS");
 
 #[program]
-pub mod multisig {
```

Comment:
It seems like there's a lot of changes made to upgrade the multisig example that are unrelated to the `InitSpace` macro. Perhaps this should be its own PR.